addons/account/models/account_move.py

```diff
@@ -3626,8 +3626,8 @@ def _check_reconcile_validity(self):
         for line in self:
             company_ids.add(line.company_id.id)
             all_accounts.append(line.account_id)
-            if (line.matched_debit_ids or line.matched_credit_ids) and line.reconciled:
-                raise UserError(_('You are trying to reconcile some entries that are already reconciled.'))
+            if line.reconciled:
+                raise UserError(_('You are trying to reconcile some entries that are already reconciled!'))
```

Comment:
then update `.pot` file, but I was told this kind of change cannot be done in stable :S

Reply:
ho good Lord -_- 

the point was to change the condition, not the punctuation of the error message. I didn't even noticed it changed... let me check that

Reply:
fixed!

thanks for noticing :+1: